components/ConnectToDiscordModal.tsx

```diff
@@ -52,11 +53,9 @@ const ConnectToDiscordModal: React.FC<ConnectToDiscordModalProps> = ({
             Connect Now
           </button>
         </NavLink>
-        <LogoutContainer>
-          <div className="text-center">
-            <a href="#">No thanks, I&apos;ll study on my own.</a>
-          </div>
-        </LogoutContainer>
+        <div className="text-center" onClick={() => close()}>
```

Comment:
The only thing the onClick callback is doing is calling the close function, so you can just pass the close function as the value. 
```
onClick={close}
```

Reply:
Hmm this gives me the following TypeScript error:

Type 'Function' is not assignable to type 'MouseEventHandler<HTMLDivElement>'.
  Type 'Function' provides no match for the signature '(event: MouseEvent<HTMLDivElement, MouseEvent>): void

Reply:
Ah, you are right -- the first param of onclick event is the `event` variable, so () => close() makes sense.